### PR TITLE
Convert to Python3

### PR DESCRIPTION
**Pull request by ralic**
using : find . -name '*.py' | xargs 2to3 -w

### CODE REVIEW

File: components/core/ConfReader.py.bak [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import os
+import inspect
+import json
+import sys
+import platform
+
+
+if platform.system() == 'Linux':
+    configdir = os.path.expanduser('~') + '/.config/bassa/'
```
Comment (rehrumesh) [inline components/core/ConfReader.py.bak]: What if check linux or Darwin?

File: components/core/DownloadDaemon.py.bak [hunk 2]
```diff
@@ -0,0 +1,225 @@
+import threading
```
Comment (rehrumesh) [inline components/core/DownloadDaemon.py.bak]: Do we need to have backup files in the source?

**Comment by rehrumesh** [comment]
@ralic Please review the comments. Thanks for supporting Bassa.

File: components/core/DiskMan.py [hunk 3]
```diff
@@ -22,7 +22,7 @@ def remove_files(days, rate):
     time_then=time_now-(days*SECS_PER_DAY)
     files=get_to_delete(time_then, rate)
     if verbose:
-        print("files to be removed", files)
+        print(("files to be removed", files))
```
Comment (ralic) [inline components/core/DiskMan.py]: Thanks, you are right!

File: components/core/DiskMan.py.bak [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import os
```
Comment (ralic) [inline components/core/DiskMan.py.bak]: It shall be removed.

**Comment by rehrumesh** [comment]
LGTM. cc @manushanga

**Comment by agentmilindu** [comment]
@malithsen @rehrumesh Can we merge this PR?

**Comment by agentmilindu** [comment]
@vivonk We should close this or we can merge this?

**Comment by kmehant** [comment]
@agentmilindu we shall **close** this PR
1. Pointing to old master branch 
2. Has some conflicts 
3. Has very little use as per the file changes.

File: components/core/ConfReader.py.bak (hunk 1 above)
Comment (JaDogg) [reply components/core/ConfReader.py.bak]: or not "Windows"?

**Comment by vivonk** [comment]
Better to close @agentmilindu @kmehant :)

**Comment by JaDogg** [comment]
Close it.

**Pull request closed without merging**